web/app/charts/MapChart.js

```diff
@@ -22,11 +22,18 @@ bluewave.charts.MapChart = function(parent, config) {
             left: 82
         }
     };
-    var svg, mapArea; //d3 elements
+    var centeringConfig = {
```

Comment:
We probably don't need this (see comments below)

Reply:
fixed with the chart config change.